### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by gianluigi1961** [OrchardCMS/OrchardCore#13573] [General]
a client of mine has a project in asp.net 4.5 that uses the orchard framework, i was able to start the orchard project but after logging in, done with orchard, it should redirect to the routes of the custom project but orchard can't find the routes
anyone who can help me?

**Comment by hishamco** [comment]
If your custom project is built as an OC module it should be discovered if you reference it in the host app. If it's not an OC module you can register your routes in the host app

**Reply by ns8482e** [reply]
@hishamco  he mentioned asp.net 4.5 so i guess it's not Orchard Core its Orchard 1.x

**Reply by gianluigi1961** [reply]
yes, it's not Core version is the old version before Core

**Reply by hishamco** [reply]
@gianluigi1961 please file the issue in https://github.com/OrchardCMS/Orchard
